contracts/JBController.sol

```diff
@@ -477,28 +502,31 @@ contract JBController is IJBController, JBTerminalUtility, JBOperatable, Reentra
     returns (uint256 beneficiaryTokenCount)
   {
     // Can't send to the zero address.
-    require(_reservedRate == 10000 || _beneficiary != address(0), '0x2f: ZERO_ADDRESS');
+    if (_reservedRate != MAX_TOKEN_RATE && _beneficiary == address(0)) {
+      revert JBErrors.ZERO_ADDRESS();
+    }
 
     // There should be tokens to mint.
-    require(_tokenCount > 0, '0x30: NO_OP');
+    if (_tokenCount == 0) {
```

Comment:
Question, should this be <= 0? I'm not sure.

Reply:
_tokenCount is an uint, ==0 is enough